supervisor/http_client.py

```diff
@@ -46,7 +50,7 @@ def __init__(
         ):
         asynchat.async_chat.__init__(self, conn, map)
         self.listener = listener
-        self.user_agent = 'Supervisor HTTP Client'
+        self.user_agent = SUPERVISOR_USER_AGENT
```

Comment:
This seems like a nice change but is it necessary in this PR? I'm wondering if maybe this should be a separate PR?